performance-tests/DCPS/TCPProfilingTest/DataReaderListener.cpp

```diff
@@ -10,32 +10,24 @@ extern long subscriber_delay_msec; // from common.h
 
 int read (::DDS::DataReader_ptr reader, bool useZeroCopy)
 {
-  // TWF: There is an optimization to the test by
-  // using a pointer to the known servant and
-  // static_casting it to the servant
   ::profilingTest::testMsgDataReader_var var_dr
     = ::profilingTest::testMsgDataReader::_narrow(reader);
 
-  ::profilingTest::testMsgDataReader_ptr pt_dr = var_dr.ptr();
-  ::profilingTest::testMsgDataReaderImpl* dr_servant =
-      dynamic_cast< ::profilingTest::testMsgDataReaderImpl*>(pt_dr);
-
   if (subscriber_delay_msec)
     {
       ACE_Time_Value delay ( subscriber_delay_msec / 1000,
                             (subscriber_delay_msec % 1000) * 1000);
       ACE_OS::sleep (delay);
     }
 
-  const ::CORBA::Long max_read_samples = 100;
+   ::CORBA::Long const max_read_samples = 100;
```

Comment:
Seem comments in other pull request about this one... let's not introduce changes that don't change anything.